test-fixtures/bundle-project/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
```

Comment:
Good catch, these were for the tests that I'd gone in to fix originally. Looks like the fixtures got sucked into this PR without the tests. Pulling these out...